waku/waku_api/rest/doc/schemas/apitypes.yaml

```diff
@@ -0,0 +1,189 @@
+WakuPeer:
+  type: object
+  required:
+    - multiaddr
+    - protocols
+  properties:
+    multiaddr:
+      type: string
```

Comment:
Also, this multiaddr should be an array of multiaddresses for the peer...as a peer can have more than 1 multiaddress advertising.

Reply:
Implemented structure here for reference
https://github.com/waku-org/go-waku/blob/9000314447e3450754dbc667a2e2049accb758d1/cmd/waku/server/rest/admin_api.yaml#L55-L74


Haven't added shardInfo yet, but will do.